/**:
  ros__parameters:
    frequency: 10.0  # Hz
    magnetic_declination_radians: 0.0
    yaw_offset: 1.5707963  # PI/2 (IMU 0 = north)
    zero_altitude: true
    publish_filtered_gps: true
    broadcast_utm_transform: true
    use_odometry_yaw: false
    wait_for_datum: false  # Use first GPS fix automatically
    datum_latitude: 0.0
    datum_longitude: 0.0
    datum_yaw: 0.0
